userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -889,92 +889,203 @@ To delete a payment method:
 
 === Invoices
 
-*_IN PROCESS_*
+This section helps you become familiar with the Invoices and Invoice Details pages.
 
-This area of Kaui lists the invoices applied to the account.
+==== Invoice Page
 
-[[more description of screen and features]]
+The Invoices page lists the invoices for the currently selected account.
 
-From this screen, you can select an invoice and make a payment on it ...
+image::Invoices.png[width=650,align="center"]
 
-==== About Automatic Invoice Payments
+Kill Bill generates invoices automatically based on the customer's subscriptions. However, you can manually create an invoice by <<_create_charge, creating a charge>> on the customer's account.
 
-Kill Bill generates an invoice automatically if:
+Click on the invoice number to open the Invoice Details page.
 
-* An account has an active, recurring subscription _and_
-* An account has a default and valid payment method _or_ you add a charge to the account.
+==== Invoice Details Page Layout
 
-Kill Bill _does not_ automatically generate an invoice payment if:
+The Invoice Details page displays information about the invoice, such as the date it was generated, amount, and so forth.
 
-* No default payment method exists for the account
-* The default payment method is invalid (e.g., expired credit card)
-* The `AUTO_PAY_OFF` or `MANUAL_PAY` https://docs.killbill.io/latest/userguide_subscription.html#components-tag[system tags] are defined for the acocunt
+image::Invoice_Detail.png[width=650,align="center"]
 
-MORE ON THIS? --> An example of this would be a check payment from the customer.
+On the Invoice Detail page, you can:
 
-==== Make a Payment on an Invoice
+* Make a payment against an invoice
+* Write off or void an invoice
+* See the payment details (if the invoice was paid)
 
-To make a manual payment against an invoice:
+[NOTE]
+*Note:* You can perform two other invoice-related tasks on the Accounts page: <<Generate a Dry-Run Invoice>> and <<Pay All Unpaid Invoices>>.
 
-1. Open the account on the Account page.
-2. Click on Invoices. Kaui displays the Invoices page:
+==== Open Invoice Details
 
-image::Invoices.png[width=650,align="center"]
+To get to the Invoice Details page:
 
-[start=3]
-. Click on the number of the invoice that needs to be paid off. This opens the invoice detail.
+. Open an account.
+. Click on the *Invoices* sub-menu.
+. Select an invoice number.
+
+[NOTE]
+*Note:* To see any associated payment details, you may need to scroll down the page.
+
+==== Make a Payment on an Invoice
+
+To make a manual payment against an invoice:
+
+1. <<Open Invoice Details, Open the Invoices Details page>> for the invoice.
 
 image::Invoice_Detail.png[width=650,align="center"]
 
-[start=4]
+[start=2]
 . Hover over the credit card icon and select *Make Payment*.
 

```

Comment:
I believe the credit card icon is not displayed if the invoice is already paid for.  Do you want to mention this here?

Reply:
Fixed. See lines 927 & 928.

Reply:
Looks good!